transport/tlsfrag/stream_dialer.go

```diff
@@ -78,3 +78,25 @@ func WrapConnFunc(base transport.StreamConn, frag FragFunc) (transport.StreamCon
 	}
 	return transport.WrapConn(base, base, w), nil
 }
+
+// NewFixedBytesStreamDialer is a [transport.StreamDialer] that modifies the [TLS Client Hello] message. It splits the
```

Comment:
I think this is more accurate?
```suggestion
// NewFixedBytesStreamDialer is a [transport.StreamDialer] that fragments the handshake TLS record. It splits the
```